pymc/distributions/continuous.py

```diff
@@ -2326,23 +2326,17 @@ def dist(cls, alpha=None, beta=None, mu=None, sigma=None, sd=None, *args, **kwar
         alpha = at.as_tensor_variable(floatX(alpha))
         beta = at.as_tensor_variable(floatX(beta))
 
-        # m = beta / (alpha - 1.0)
-        # try:
-        #     mean = (alpha > 1) * m or np.inf
-        # except ValueError:  # alpha is an array
-        #     m[alpha <= 1] = np.inf
-        #     mean = m
-
-        # mode = beta / (alpha + 1.0)
-        # variance = at.switch(
-        #     at.gt(alpha, 2), (beta ** 2) / ((alpha - 2) * (alpha - 1.0) ** 2), np.inf
-        # )
-
         assert_negative_support(alpha, "alpha", "InverseGamma")
         assert_negative_support(beta, "beta", "InverseGamma")
 
         return super().dist([alpha, beta], **kwargs)
 
+    def get_moment(rv, size, alpha, beta):
+        m = beta / (alpha - 1.0)
+        if not rv_size_is_none(size):
+            m = at.full(size, m)
+        return at.switch(alpha > 1, m, np.inf)
```

Comment:
We should either use the mode as the moment or switch between the mode and the mean, so that we don't have to return an infinite value. The goal of `get_moment` is to obtain a value that is always okay as a starting point for that distribution.

Reply:
Okay, I think I fixed it now! Let me know if I need to make further changes